microhapulator/api.py

```diff
@@ -89,16 +89,18 @@ def interlocus_balance(
             data.to_csv(tfile, index=False, header=False)
             run(["termgraph", tfile])
     if tofile:
+        backend = matplotlib.get_backend()
+        plt.switch_backend("Agg")
```

Comment:
Adding this to all plotting functions to avoid the excessive spilling of all kinds of messy cyber blood all over the terminal.